.ado/pipelines/templates/stages-loadtest-azure.yaml

```diff
@@ -138,7 +157,7 @@ stages:
 
       # Create config csv for Azure Load Test
```

Comment:
should we then just drop this task, too?

Reply:
was already dropped just not synced